### PR TITLE
Base ml ui

### PR DESCRIPTION
**Pull request by bago2k4**
Cards:
 - https://trello.com/c/C4rjx0BB
 - https://trello.com/c/i4voGPlF

Related PRs:

- [x] Auth: https://github.com/open-company/open-company-auth/pull/19
- [X] Api: https://github.com/open-company/open-company-api/pull/84
- [X] Lib: https://github.com/open-company/open-company-lib/pull/13
- [X] Interaction: https://github.com/open-company/open-company-interaction/pull/4

### CODE REVIEW

**Comment by belucid** [comment]
@bago2k4 don't think we want `ML` in the path of the codebase as was done for the images paths.

File: scss/modules/_colors.scss [hunk 1]
```diff
@@ -21,13 +21,22 @@ $oc_gray_2: #D8D8D8;
...
 $oc_red_regular: #E04B53;
 $oc_red_light: #EC7A8D;
 // Charts
 $oc_chart_blue: #4EE4D6;
 $oc_chart_green: #4de148;
-$oc_chart_red: #D72A46;
\ No newline at end of file
+$oc_chart_red: #D72A46;
+
+$cr_text_blue: #4c67fc;
```
Comment (belucid) [inline scss/modules/_colors.scss]: not sure `cr` is quite as clear as `oc` was. Maybe just `carrot_...` is better here.

File: resources/public/img/ML/about_carrot.svg [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (belucid) [inline resources/public/img/ML/about_carrot.svg]: Let's not have `ML` in the image paths.

File: scss/modules/_colors.scss [hunk 3]
```diff
@@ -21,13 +21,22 @@ $oc_gray_2: #D8D8D8;
...
 $oc_gray_5: #4E5A6B;
-$oc_gray_7: #0A0A0A;
-$oc_gray_8: #F8F8F8;
+$oc_gray_7: #637282;
+$oc_gray_8: #FBFAF7;
 // Reds
 $oc_red_dark: #9E001A;
 $oc_red_regular: #E04B53;
 $oc_red_light: #EC7A8D;
 // Charts
```
Comment (belucid) [inline scss/modules/_colors.scss]: Do we still need these chart colors?

File: scss/partials/_bw_boards_list.scss [hunk 4]
```diff
@@ -2,56 +2,86 @@ div.left-boards-list {
   float: left;
```
Comment (belucid) [inline scss/partials/_bw_boards_list.scss]: What does `bw` mean in the file name `_bw_boards_list`?

File: site/oc/core.clj [hunk 5]
```diff
@@ -4,7 +4,7 @@
             [hiccup.page :as hp]
             [environ.core :refer (env)]))
 
-(def contact-email "hello@opencompany.com")
+(def contact-email "hello@carrot.io")
```
Comment (belucid) [inline site/oc/core.clj]: Let's keep this as opencompany.com for now. We haven't set up email on carrot.io yet.

File: site/oc/core.clj [hunk 6]
```diff
@@ -61,7 +61,7 @@
           [:li
               [:a.navbar-item {:href "/about"} "About"]]
           [:li.mobile-only
-            [:a.navbar-item.contact {:href "mailto:hello@opencompany.com"} "Contact"]]]]]])
+            [:a.navbar-item.contact {:href "mailto:hello@carrot.io"} "Contact"]]]]]])
```
Comment (belucid) [inline site/oc/core.clj]: how come the constant isn't used here?

File: site/oc/core.clj [hunk 7]
```diff
@@ -83,7 +83,7 @@
       [:li.web-only
         [:a {:href "/about"} "About"]]
       [:li
-        [:a.contact {:href "mailto:hello@opencompany.com"} "Contact"]]
+        [:a.contact {:href "mailto:hello@carrot.io"} "Contact"]]
```
Comment (belucid) [inline site/oc/core.clj]: how come the constant isn't used here?

File: src/oc/web/actions.cljs [hunk 8]
```diff
@@ -1103,4 +1104,8 @@
 
 (defmethod dispatcher/action :ws-interaction/reaction-delete
   [db [_ interaction-data]]
-  (update-reaction db interaction-data false))
\ No newline at end of file
+  (update-reaction db interaction-data false))
+
+(defmethod dispatcher/action :trend-bar-status
```
Comment (belucid) [inline src/oc/web/actions.cljs]: Needs added to the README.

**Comment by belucid** [comment]
@bago2k4 

- [ ] Only have the thin orange line separating the header for now. Don't have it expand on hover.
- [ ] The + buttons in the left nav. should have the same hover styling that the "Start Writing" button has.
- [ ] Fonts on "Hi Sean!" and "You're an Admin" are wrong.
- [ ] Fonts on Board names (not BOARDS) is wrong.
- [ ] Should be dividers in the avatar drop down menu
- [ ] Couldn't get to User Profile from the menu, nothing happens.

Just a note, no action at the moment, when we do add the expand on hover of the orange line back in, it shouldn't cause the content below to jump, should just take up more of the white space that's already there. Won't change the actual space unless they click expand.

**Comment by belucid** [comment]
@bago2k4 

FWIW, `OCWebForceRefreshToken()` didn't work for me:

http://take.ms/H7W78

**Comment by belucid** [comment]
@bago2k4 

We need to guess the user's initial TZ from the browser:
http://momentjs.com/timezone/docs/#/using-timezones/guessing-user-timezone/

**Comment by belucid** [comment]
@bago2k4 

To the concern about too many TZ's, let's just elevate these 4:

US/Eastern
US/Central
US/Mountain
US/Pacific

Put them at the top of the dropdown list, then a divider, then the alphabetical list (repeat them in the alphabetical list).

**Comment by bago2k4** [comment]
@belucid all fixed. I'm missing only the font fix as you can see above but i need to do that in `marketing-site` branch since i have added the Avenir font only there.

**Comment by belucid** [comment]
@bago2k4 I meant that it was using OpenSans rather than Muli (not Avenir). But we can leave the font stuff for now.

**Pull request merged**